docs/user/Alerts.md

```diff
@@ -13,16 +34,16 @@ persisted for 10 minutes or longer.
 * The node was deleted from the cluster via the api, but the machine still exists
 
 ### Resolution
-If the machine never became a node, consult the machine troubleshooting guide.
+If the machine never became a node, consult the [machine troubleshooting guide](TroubleShooting.md).
 If the node was deleted from the api, you may choose to delete the machine object, if appropriate.  (FYI, The machine-api will automatically delete nodes, there is no need to delete node objects directly)
 
-## MachineWithNoRunningPhase
+## MachineNotStarting
```

Comment:
Perhaps a better name would be `MachineNotYetRunning`? I feel NotStarting implies that we have failed to even start the VM, but this error case is actually much deeper than that and the VM could actually be starting up fine

Reply:
Done.